dub.json

```diff
@@ -5,7 +5,15 @@
     "homepage": "https://github.com/blachlylab/dhtslib",
     "license": "MIT",
     "dependencies": {
+        "dhtslib:coordinates": "*"
     },
+    "subPackages": [
+        "./coordinates/"
+    ],
+    "sourcePaths": [
+        "./source/",
+        "./coordinates"
```

Comment:
Is this required if you include the "dependencies" above? I don't think so, based on documentation examples at https://dub.pm/package-format-json.html

I believe it (`"./coordinates"`) needs to be removed

Reply:
Yes it is required. This [post](https://forum.dlang.org/thread/tjvachjznbivzofpiwjw@forum.dlang.org) explains how subpackages are a bit complicated. I used the [repo](https://github.com/aliak00/dub-subpackages) they discussed as a template. Removing this line from the file results in a compile error because the compiler can't find `coordinates`.

Reply:
*sigh* dub documentation strikes again